pkgs/development/ocaml-modules/rdbg/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, buildDunePackage, fetchurl, ocaml, findlib, num, lutils, ounit, ledit, git}:
+
+buildDunePackage rec {
+  pname = "rdbg";
+  version = "1.196.12";
+
+  useDune2 = true;
+
+  src = fetchurl {
+    url = "http://www-verimag.imag.fr/DIST-TOOLS/SYNCHRONE/pool/rdbg.1.196.12.tgz";
+    sha512 = "8e88034b1eda8f1233b4990adc9746782148254c93d8d0c99c246c0d50f306eeb6aa4afcfca8834acb3e268860647f47a24cc6a2d29fb45cac11f098e2ede275";
+  };
+
+  buildInputs = [
+    ocaml
+    findlib
+    ledit
+    num
+    lutils
```

Comment:
The `lutils` library should be propagated.

Reply:
Added `lutils` to `propagatedBuildInputs` as suggested

Reply:
Now that it’s in `propagatedBuildInputs`, you can remove it from the `buildInputs`.